lib/compress/zstd_compress.c

```diff
@@ -983,6 +983,7 @@ static size_t ZSTD_resetCCtx_internal(ZSTD_CCtx* zc,
 
     if (params.ldmParams.enableLdm) {
         /* Adjust long distance matching parameters */
+        params.ldmParams.windowLog = params.cParams.windowLog;
```

Comment:
Do you foresee, one day, the possibility for these 2 values to be different ?
Should they always be identical ?

Reply:
Thats a good point. I don't see a use for them to be different.

I was initially thinking each individual chunk in zstdmt would have a smaller window log, but it will be effectively limited using the chunk size and overlap log.